components/portal/main/controllers.js

```diff
@@ -33,6 +33,29 @@ define(['angular', 'require'], function(angular, require) {
       layoutMode: 'list', // other option is 'widgets
     };
 
+    /**
+     * Set Document title.
+     * Asks mainService what the document title ought to be and
+     * sets the document title to that value.
+     * @param {string} pageTitle - Optional, name of specific page viewed.
```

Comment:
:art: jsDoc syntax for optional is
```javascript
/** @param {string} [pageTitle] - name of specific page viewed
```

reference: http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values